approach_mission.md

```diff
@@ -0,0 +1,127 @@
+Yes, he can! So it will look like this:
+
+![](2.jpg)
+
+And if our author is particularly diligent, he can write even a million articles (theoretically of course…), so this will bring us to the second important question, we call it “the million question”: 
+2. Is the entity “author” can refer to million entities “article”?
+No, the answer is definitely not. To understand that, we need to understand the meaning of reference. When object A refer to object B it means object A “knows” about object B, and this “knowing” became part of the information that object A holds.  So when we call object A from the Database, it will brings us also the information of all the “B”s objects that it refer to. So if there are milion “B”s objects, the systems brings them all. This is a very heavy task for the system to deal with and needs a lot of memory resources … We want to make it easier for the system and not make it difficult for her. 
+
+So because we don’t want “author” refer to million entities “article”, the “article” is refer to “author”. 
+It will look like this:
+
+![](3.jpg)
+
+
+Now let’s add more details to our story:
+
+**Author writes articles in various topics.**
+
+Now we have another entity “topic”. How does “topic” relate to the others entities?
+Article written on a particular topic, then the ״article״ and the “topic” have a relationship.
+
+![](4.jpg)
+
+
+And who refer to whom? (what is the arrow direction?)
+Let’s ask the “million question” - 
+Is one topic can has more than one article? Yes.
+Is one article can written on more than one topic? Theoretically yes, but for the practice we decide that every article belongs only to one topic, so the answer to this question is No.
+
+
+//pic5a 
+![](5a.jpg)
+
+
+//pic5b
+![](5b.jpg)
+
+
+Our story continue - 
+
+**We going to build a Premium Website, contains articles. People can signup and register to one or more topic that interest them. While they register to a topic, they can read articles belong to it.**
+
+Now we have new entity: People.  Let's call it “User”.
+
+Is “user” has relationship with “topic”?
+
+Well, user can register to many topics and many users can register to the same topic, so we will get something like this:
+
+![](6.jpg)
+
+
+Try to answer the “million question” for this case… a bit complicated, huh?! It can be a million users and a million topics...
+So, to make it simple, we can add more entity - let’s call it “Membership”, and it will represent a specific register of user to a topic.
+
+![](7.jpg)
+
+In that case “Membership” refer to “Topic” (because we don’t want “Topic” refer to million “Membership”) and ““Membership” also refer to “User” (because we don’t want “User” refer to million “Membership”). It looks like this:
+
+![](88.jpg)
+
+
+
+Let us continue with our story - 
+
+**Registration expired after a year. The user will get a reminder emails - 3 month, 1 month and 1 day before expired, so he can renew his registration.** 
```

Comment:
I usually prefer to highlight just a word or two. Once it's a whole sentence it's hard to follow on what's really important.